docker-compose.yml

```diff
@@ -23,6 +23,8 @@ services:
       - ALLOWED_HOSTS=["*"]
       - CORS_ORIGIN_WHITELIST=('localhost:3000',)
       - GATHER_CASES_CRON_TIME_SECONDS=1440
+      - FACEBOOK_APP_ID=108193869861396
+      - FACEBOOK_APP_SECRET=6c7e431f5dda1d4075f027fe0b667ee8
```

Comment:
It is currently in development mode. But I just created it for dev/testing purposes and I think it would be better that either one of you would later create another Facebook app that would be used.